teps/XXXX-task-execution-status-at-runtime.md

```diff
@@ -0,0 +1,115 @@
+---
+title: task-exec-status-at-runtime
+authors:
+  - "@pritidesai"
+creation-date: 2020-10-15
+last-updated: 2020-10-19
+status: proposed
```

Comment:
with all the review + discussion we've done, i wonder if this is a case where we could go straight to "implementable"?

Reply:
Changed it to `implementable`